detox/e2e/support/ui/screen/login.ts

```diff
@@ -60,9 +60,19 @@ class LoginScreen {
 
     login = async (user: any = {}) => {
         await this.toBeVisible();
-        await this.usernameInput.replaceText(user.username);
-        await this.passwordInput.replaceText(user.password);
-        await this.signinButton.tap();
+        await this.usernameInput.typeText(`${user.newUser.email}\n`);
+        await this.passwordInput.typeText(`${user.newUser.password}\n`);
+
+        // await this.signinButton.tap();
```

Comment:
If we don't tap the button, we shouldn't be able to log in, right? Or the newline in line 64 does the trick?

Reply:
This was a bit weird.
earlier `this.passwordInput.replaceText` was used.
typeText is how user will type using keyboard and it will also trigger input callbacks.

the `\n` was doing the trick of pressing on `next` button, else the keyboard covers elements, and it was extra step of handlign keyboard that I avoided.